lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -42,19 +43,77 @@ class DSLocationMessageBubble extends StatelessWidget {
 
     final lat = double.tryParse(latitude);
     final long = double.tryParse(longitude);
-
     final hasValidCoordinates = lat != null && long != null;
-    return GestureDetector(
-      onTap: () async {
-        final availableMaps = await MapLauncher.installedMaps;
+    List<AvailableMap>? availableMaps;
+
+    void initMaps() async {
+      availableMaps = await MapLauncher.installedMaps;
+    }
+
+    initMaps();
 
-        if (hasValidCoordinates) {
-          await availableMaps.first.showMarker(
-            coords: Coords(lat, long),
-            title: title ?? '',
-          );
-        }
-      },
+    void openMapList() {
+      if (availableMaps!.isNotEmpty && hasValidCoordinates) {
+        showModalBottomSheet(
+            showDragHandle: true,
+            shape: const RoundedRectangleBorder(
+              borderRadius: BorderRadius.vertical(
+                top: Radius.circular(
+                  30.0,
+                ),
+              ),
+            ),
+            context: context,
+            builder: (context) {
+              return SafeArea(
+                child: SingleChildScrollView(
+                  child: Wrap(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.symmetric(
+                          horizontal: 16.0,
+                          vertical: 8.0,
+                        ),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            DSHeadlineLargeText(
+                              "Selecione uma ação",
+                              fontWeight: FontWeight.bold,
+                            ),
+                            IconButton(
+                              onPressed: () => Navigator.pop(context),
+                              icon: const Icon(Icons.close),
+                            )
+                          ],
+                        ),
+                      ),
+                      const Divider(
+                        thickness: 2.0,
+                      ),
+                      for (var map in availableMaps!)
```

Comment:
My suggestion is to eliminate this ```for``` and use ```ListView.builder``` to iterate across all available maps.

Reply:
Done!